### PR TITLE
Add empty function to glb in Cpp by passing an empty function

### PR DESCRIPTION
**Pull request by juliuspfadt**

### CODE REVIEW

File: ToDo.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+To Do:
```
Comment (vandenman) [inline ToDo.txt]: redundant file? Also, I'd add things like `# TODO:` directly in the source code. Most editors have highlighting for that and it's more easily searchable.

File: src/customsdpCpp.cpp [hunk 2]
```diff
@@ -446,6 +447,11 @@ int custom_sdpCpp(
            diagO,bestx,besty,bestz,Zi,O,rhs,dZ,dX,dy,dy1,Fp,
            printlevel,params);
 
+		if (i % 500 == 0)
```
Comment (vandenman) [inline src/customsdpCpp.cpp]: so this is definitely fine, I just wonder if the 500 shouldn't be a function of the number of samples. Maybe you want to run the callback after the maximum of 100 iterations or 1% of the samples? So something like `std::min(int(0.01 * total_no_samples), 100);`?

Comment (juliuspfadt) [reply src/customsdpCpp.cpp]: I guess I could just make it so that I have only five ticks for the progress bar, irrespective of no samples

Comment (juliuspf) [reply src/customsdpCpp.cpp:450]: that also depends on the number if chains though, and that is not really passed to the glb.

Comment (juliuspfadt) [reply src/customsdpCpp.cpp]: I altered it, so that there are always 4 ticks per chain no matter how many samples.

**Pull request merged**